cluster/saltbase/salt/etcd/etcd.manifest

```diff
@@ -1,3 +1,5 @@
+{% set num_nodes = salt['grains.get']('num_cpus', '') %}
```

Comment:
Oops :) I copy&pasted this line from an example and forgot to change the variable name. Thanks!